data/companies.yml

```diff
@@ -209,8 +209,8 @@ integrations:
       - "Provides complete layer 7 protection for microservices architecture applications."
   - name: "Solo.io"
     logo: "/logos/solo.png"
-    url: "https://www.solo.io/"
-    description: "Solo.io offers Gloo Mesh, building on Istio, with expert support."
+    url: "https://webassemblyhub.io/"
+    description: "Solo.io offers WebAssembly Hub to share Wasm filters for Envoy/Istio, with expert support."
```

Comment:
Sorry, I didn't see this come through

Please title this "WebAssemblyHub" with appropriate logo; and don't include Solo branding in the description.  This is a place for community, not marketing; you've got marketing on your other tabs.